satellite/peer.go

```diff
@@ -319,67 +304,25 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 			Node: pb.Node{
 				Id: peer.ID(),
 				Address: &pb.NodeAddress{
-					Address: config.ExternalAddress,
+					Address: c.ExternalAddress,
 				},
 			},
-			Type: pb.NodeType_SATELLITE,
-			Operator: pb.NodeOperator{
-				Email:  config.Operator.Email,
-				Wallet: config.Operator.Wallet,
-			},
-			Version: *pbVersion,
-		}
-
-		{ // setup routing table
-			// TODO: clean this up, should be part of database
-			log.Debug("Setting up routing table")
-			bucketIdentifier := peer.ID().String()[:5] // need a way to differentiate between nodes if running more than one simultaneously
-			dbpath := filepath.Join(config.DBPath, fmt.Sprintf("kademlia_%s.db", bucketIdentifier))
-
-			if err := os.MkdirAll(config.DBPath, 0777); err != nil && !os.IsExist(err) {
-				return nil, err
-			}
-
-			dbs, err := boltdb.NewShared(dbpath, kademlia.KademliaBucket, kademlia.NodeBucket, kademlia.AntechamberBucket)
-			if err != nil {
-				return nil, errs.Combine(err, peer.Close())
-			}
-			peer.Kademlia.kdb, peer.Kademlia.ndb, peer.Kademlia.adb = dbs[0], dbs[1], dbs[2]
-
-			peer.Kademlia.RoutingTable, err = kademlia.NewRoutingTable(peer.Log.Named("routing"), self, peer.Kademlia.kdb, peer.Kademlia.ndb, peer.Kademlia.adb, &config.RoutingTableConfig)
-			if err != nil {
-				return nil, errs.Combine(err, peer.Close())
-			}
-
-			peer.Transport = peer.Transport.WithObservers(peer.Kademlia.RoutingTable)
+			Type:     pb.NodeType_SATELLITE,
+			Operator: pb.NodeOperator{},
```

Comment:
why not pass in the config.operator.email/wallet?

Reply:
I think we can actually remove this

Reply:
I guess we could, but it's not used anywhere. I'll see if I can remove that whole field from this setup